openlibrary/templates/lists/list_follow.html

```diff
@@ -0,0 +1,61 @@
+$def with(lists, limit, user_key)
+
+$def render_follow_button(owner_username, is_subscribed):
+  $ request = "/people/" + owner_username
+  $:macros.Follow(owner_username, following=is_subscribed, request_path=request)
+
+$def list_card(list, owner, own_list):
+    $ has_owner = owner
+    $ cached_info = list.get_patron_showcase()
+    $ count = cached_info["count"]
+    <div class="list-follow-card">
+        <a class="list-follow-card__header" href="$list.get_url()">
+             <div class="list-follow-card__title">$cached_info["title"]</div>
+             <div class="list-follow-card__num-books">
+                 $ungettext("%(count)d book", "%(count)d books", count, count=count)
+             </div>
+        </a>
+        <a class="list-follow-card__covers" href="$list.get_url()">
+           $for img_url in cached_info["covers"]:
+               $if img_url:
+                   $ img_url = img_url.replace("-S.jpg", "-M.jpg")
+               $else:
+                   $ img_url = '/images/icons/avatar_book-sm.png'
+               <img src="$img_url" loading="lazy" width="80"/>
```

Comment:
Move vars like `$owner_username` to the top of the method so they can be referenced more easily, e.g. in adding alt text:

```suggestion
               <img alt="$cached_info['title'] - a list by $('@' + owner_username)" src="$img_url" loading="lazy" width="80"/>
```